clients/devtools-report-assets.js

```diff
@@ -14,23 +14,23 @@
 /* global Runtime */
```

Comment:
```suggestion
/* global globalThis */
```

to make eslint happy